apps/mull-api/src/app/event/event.resolver.ts

```diff
@@ -34,6 +34,14 @@ export class EventResolver {
     return this.eventService.findJoinedEvents(id);
   }
 
+  @Query(/* istanbul ignore next */ () => Boolean)
+  async participatingEvent(
+    @Args('eventId', { type: /* istanbul ignore next */ () => Int }) eventId: number,
+    @Args('userId', { type: /* istanbul ignore next */ () => Int }) userId: number
+  ) {
+    return !!(await this.eventService.findJoinedEvent(eventId, userId));
```

Comment:
The naming of these methods is super confusing. Also, try to follow the general naming you see in #147 .

The resolver method could be called `isParticipant`, and the service name could be `isUserAttendingEvent`